src/main/webapp/app/components/tabs/CurationToolsTab.tsx

```diff
@@ -1,105 +1,159 @@
-import { PAGE_ROUTE } from 'app/config/constants/constants';
-import { Gene, Meta } from 'app/shared/model/firebase/firebase.model';
-import GeneSelect from 'app/shared/select/GeneSelect';
-import { getFirebasePath } from 'app/shared/util/firebase/firebase-utils';
 import { componentInject } from 'app/shared/util/typed-inject';
 import { IRootStore } from 'app/stores';
 import React, { useEffect, useRef, useState } from 'react';
-import { FaExclamationCircle } from 'react-icons/fa';
-import { Button, Col, Row } from 'reactstrap';
-import './curation-tools-tab.scss';
+import { Button, Row } from 'reactstrap';
+import { FaRegCheckCircle } from 'react-icons/fa';
+import { FaRegCircleXmark } from 'react-icons/fa6';
 import { notifyError } from 'app/oncokb-commons/components/util/NotificationUtils';
-import { isPromiseOk } from 'app/shared/util/utils';
-import { getErrorMessage } from 'app/oncokb-commons/components/alert/ErrorAlertUtils';
-import { observer } from 'mobx-react';
+import { IGene } from 'app/shared/model/gene.model';
+import _ from 'lodash';
+import { IFlag } from 'app/shared/model/flag.model';
 
-function CurationToolsTab({ addMetaListener, metaData, createMetaGene, deleteMetaGene, createGene, deleteGene }: StoreProps) {
-  const selectedGene = useRef<string>(null);
-  const [createButtonDisabled, setCreateButtonDisabled] = useState(true);
-  const [showGeneExistsWarning, setShowGeneExistsWarning] = useState(false);
+export type ReleaseGeneTestData = {
+  passed: boolean;
+  text: string;
+  type: 'optional' | 'required';
+};
+
+export function CurationToolsTab({ metaList, addMetaListListener, gene, geneEntities, searchGenes, updateGene, searchFlags }: StoreProps) {
+  const reviewList = metaList?.[gene.name].review;
+
+  const tests: ReleaseGeneTestData[] = [
+    {
+      passed: reviewList ? !Object.keys(reviewList).some(key => key !== 'currentReviewer' && reviewList[key]) : true,
```

Comment:
Can we create a constant of `currentReviewer`? @calvinlu3 this is already related to ur work I believe on how to determine whether there is review content.